app/assets/stylesheets/photos.scss

```diff
@@ -0,0 +1,15 @@
+// Place all the styles related to the photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.likes-link {
+  text-decoration: none;
+  color: black;
+}
+
+#comment_bubble {
+  text-decoration: none;
+}
+
+heart {
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.